linera-execution/src/policy.rs

```diff
@@ -67,6 +67,8 @@ pub struct ResourceControlPolicy {
     pub maximum_bytes_read_per_block: u64,
     /// The maximum data to write per block
     pub maximum_bytes_written_per_block: u64,
+    /// The maximum size in bytes of an oracle response.
+    pub maximum_oracle_response_bytes: u64,
```

Comment:
Let's make sure all resource policy options can be set via the command line. (See https://github.com/linera-io/linera-protocol/pull/3598.)

Reply:
I was going to do this in a separate PR that would also catch these issues: #3532.

Reply:
(cc @MathieuDutSik)

Reply:
I'll wait for #3598 to be merged first, than I can rebase and fix it here as well.

Reply:
Done, thanks!